internal/pkg/deploy/cloudformation/stack/testdata/pipeline/bb_template.yaml

```diff
@@ -111,7 +111,10 @@ Resources:
         Type: LINUX_CONTAINER
         ComputeType: BUILD_GENERAL1_SMALL
         PrivilegedMode: true
-        Image: aws/codebuild/amazonlinux2-x86_64-standard:1.0
+        Image: aws/codebuild/amazonlinux2-x86_64-standard:3.0
```

Comment:
yay! Do you know which version of the AWS CLI is shipped with 3.0? Is it v1 or v2?

Reply:
Pretty sure v1 still.
v2 is shipped with Standard:5.0 https://github.com/aws/aws-codebuild-docker-images/issues/407